app/components/Footer/index.js

```diff
@@ -77,7 +71,10 @@ const Footer = (props: Props) => (
             <p>7491 Trondheim</p>
           </div>
         </Flex>
-        <Image className={styles.cooperator} src={netcompany} />
+        <Image
+          className={cx(styles.cooperator, hiddenOnDesktop)}
```

Comment:
What's the thought here? The other logo is inside a `hiddenOnMobile` div and this image gets the `hiddenOnDesktop` class. Doesn't that mean the image is shown neither on phone or on desktop?

Reply:
I believe the thought is to remove the logo in the bottom and instead have it in place of the random quoute on desktop. Due to the location of the logo being different, the components have to be in different places.